kano_settings/bluetooth_config.py

```diff
@@ -50,10 +50,10 @@ def __init__(self, device):
 
     def _set_paired_button_state(self, *dummy_args, **dummy_kwargs):
         if not self.device.connected:
-            label = 'Pair'.upper()
+            label = _("PAIR")
             colour = 'green'
         else:
-            label = 'Unpair'.upper()
+            label = _("UNPAIR")
```

Comment:
Prefer to pre-capitalise as different languages may use different characters for upper vs lower. French for example do not put the accents on capitalised letters.